prism-highlighter.html

```diff
@@ -59,7 +59,12 @@
         return;
       }
 
+      if(event.detail._highlighted === true) {
```

Comment:
This is true but then what do we do about calling `_detectLang` if `this` isn't bound? Wouldn't be great to move that into the event listener.